fix(checkout v3): UI tweaks

itsdangold opened the pull request:
Making some broad stroke changes to the new checkout UI
- making everything more "chonky"
- slightly changed app structure and wrapping
- adjustments to color, typography, spacing

itsdangold committed:
Fix layout breakpoints

itsdangold committed:
Cart styling tweaks

itsdangold committed:
Structural styling

itsdangold committed:
Typography tweaks

itsdangold committed:
Refactor plan select card styling

itsdangold committed:
Additional products styling

itsdangold committed:
Billing cycle UI tweaks and refactor

itsdangold committed:
Update PAYG table styling

itsdangold committed:
Tidy up checkout structure

itsdangold committed:
Squashed commit of the following:

commit daae74a6b1f2b636c03730266b776b48dfda1e2c
Author: isabellaenriquez <isabella.enriquez@sentry.io>
Date:   Fri Sep 12 09:20:37 2025 -0400

    add breakpoint

commit 5838176482f312060f6f1dc64b9b549090b615f6
Author: isabellaenriquez <isabella.enriquez@sentry.io>
Date:   Thu Sep 11 17:52:17 2025 -0400

    move price warning

commit 29187c01d00ca879f38cbd41b9a5913e8f947bd7
Author: isabellaenriquez <isabella.enriquez@sentry.io>
Date:   Thu Sep 11 17:28:13 2025 -0400

    feat(checkout v3): New plan feature list

itsdangold committed:
Increase chonk

itsdangold committed:
Add grid to additional products

itsdangold committed:
Remove PAYG example

itsdangold committed:
Tidy up plan card features

itsdangold committed:
Improve plan selection styling

itsdangold committed:
Reduce plan feature icon size

itsdangold committed:
Resize cart and logo

itsdangold committed:
Reserve card styling

itsdangold committed:
Cart styling tweaks

itsdangold committed:
Checkout spacing

itsdangold committed:
Chonky card interactions

itsdangold committed:
Table adjustments

isabellaenriquez commented on static/gsApp/views/amCheckout/cart.tsx:
```diff
@@ -673,29 +674,34 @@ function Cart({
+          <Flex gap="xs" align="center">
+            <OrgSlug>{organization.slug.toUpperCase()}</OrgSlug>
+            <Button
+              aria-label={`${summaryIsOpen ? 'Hide' : 'Show'} plan summary`}
```
oh shoot i forgot to wrap this in a `t` tag

itsdangold replied:
fixed

itsdangold commented on static/gsApp/views/amCheckout/index.tsx:
```diff
@@ -906,115 +897,116 @@ class AMCheckout extends Component<Props, State> {
-          <Grid
+          <Flex
+            direction="column"
+            align="start"
             gap="2xl"
-            columns={{
-              sm: 'auto',
-              md: isNewCheckout ? '3fr 2fr' : 'auto',
-              lg: '3fr 2fr',
-            }}
             maxWidth={isNewCheckout ? '1440px' : undefined}
-            padding={isNewCheckout ? '2xl' : undefined}
```
ah crap thought i fixed that, good catch

itsdangold commented on static/gsApp/views/amCheckout/steps/checkoutV3/buildYourPlan.tsx:
```diff
@@ -169,12 +169,14 @@ function AdditionalProductsSubstep({
-        onUpdate={onUpdate}
-        isNewCheckout
-      />
+      <Grid columns="1fr 1fr" gap="xl">
+        <ProductSelect
+          activePlan={activePlan}
+          formData={formData}
+          onUpdate={onUpdate}
+          isNewCheckout
+        />
+      </Grid>
```
yeah i figured this was a bandaid for now, i'll add the breakpoint too!

itsdangold committed:
Resolve comments

- add translation to cart label
- change structure to show new layout on isNewCheckout
- responsive grid

isabellaenriquez committed:
Merge branch 'master' into dang/new-checkout-ui

isabellaenriquez committed:
fix payg price table

isabellaenriquez committed:
only show chonky styles for chonk theme

isabellaenriquez committed:
only show chonky styles for chonk theme pt2

isabellaenriquez committed:
add aria attrs + fix tests

isabellaenriquez committed:
fix test
